authors/apps/articles/renderers.py

```diff
@@ -38,3 +38,33 @@ def render(self, data, media_type=None, renderer_context=None):
         })
 
 
+class ArticleReportJSONRenderer(JSONRenderer):
+    charset = 'utf-8'
+
+    def render(self, data, media_type=None, renderer_context=None):
+
+        errors = data.get('errors', None)
+
+        if errors is not None:
+
+            return super(ArticleReportJSONRenderer, self).render(data)
+
+        return json.dumps({
+            'reported': data
+        })
+
+
+class ArticleListReportJSONRenderer(JSONRenderer):
+    charset = 'utf-8'
+
+    def render(self, data, media_type=None, renderer_context=None):
+
+        errors = data.get('errors', None)
+
+        if errors is not None:
+
+            return super(ArticleListReportJSONRenderer, self).render(data)
```

Comment:
henry, you missed testing this renderer class

Reply:
I get but I tested almost  every functionality
But that line remained, however, my coverage passes